src/ivector/xvector-test.cc

```diff
@@ -0,0 +1,131 @@
+// ivector/xvector-test.cc
+
+// Copyright 2016  Daniel Povey
```

Comment:
just have your own name on this file.
only include me in the header, but since I'm an employee of JHU you'd have to put
//  Copyright 2016    Johns Hopkins University  (Author: Daniel Povey)
                    2016    David Snyder